controllers/assignments.py

```diff
@@ -529,13 +528,13 @@ def _autograde_one_q(course_name, assignment_id, sid, qname, points, deadline=No
     question = db(db.questions.name == qname).select().first()
 
     # dispatch on grading_type; if none specified, can't autograde
-    if question.autograde == 'unittest':
+    if autograde == 'unittest':
```

Comment:
If an instructor is following a grading path of chapter --> question will this still work?  Why the change from autograde to question.autograde ??